src/ownArrayList.java

```diff
@@ -0,0 +1,189 @@
+import java.util.Arrays;
+
+public class ownArrayList<T> implements List {
+
+    private final int INITIALIZE_CAPACITY = 3;
+
+    private int capacity;
```

Comment:
do we need capacity, oldCapacity and newCapacity? May we work with arraylist without them?